controllers/suite_test.go

```diff
@@ -45,6 +47,8 @@ func TestMain(m *testing.M) {
 func setup() {
 	utilruntime.Must(infrav1.AddToScheme(scheme.Scheme))
 	utilruntime.Must(clusterv1.AddToScheme(scheme.Scheme))
+	utilruntime.Must(expclusterv1.AddToScheme(scheme))
```

Comment:
Shouldn't this be `scheme.Scheme`? Maybe that's what's causing issues.

Reply:
possibly - sorry, my second commit was actually just WIP garbage commit where I tried to find the cause. Will revert those unnecessary changes.